src/webui/www/public/preferences_content.html

```diff
@@ -57,10 +57,17 @@
     <legend><input type="checkbox" id="autorun_checkbox" onclick="updateAutoRun();"/>
     <label for="autorun_checkbox">QBT_TR(Run an external program on torrent completion)QBT_TR</label></legend>
     <input type="text" id="autorunProg_txt" style="width: 400px;"/><br/>
-    <i>QBT_TR(The following parameters are supported:)QBT_TR
+    <i>QBT_TR(Supported parameters (case sensitive):)QBT_TR
     <ul>
-    <li>%f: QBT_TR(Torrent path)QBT_TR</li>
-    <li>%n: QBT_TR(Torrent name)QBT_TR</li>
+    <li>%N: QBT_TR(Torrent name)QBT_TR</li>
```

Comment:
You should put this instead, if not the translation is not working properly:
```
<li>QBT_TR(%N: Torrent name)QBT_TR</li>
```
You have to change all lines to match the exact translation => https://github.com/qbittorrent/qBittorrent/blob/master/src/gui/options_imp.cpp#L186